client/components/tinymce/plugins/contact-form/plugin.js

```diff
@@ -3,18 +3,21 @@
  */
 import tinymce from 'tinymce/tinymce';
 import i18n from 'lib/mixins/i18n';
-import ReactDom from 'react-dom';
-import ReactDomServer from 'react-dom/server';
-import React from 'react';
-
+import React, { createElement } from 'react';
+import { render, unmountComponentAtNode } from 'react-dom';
+import { renderToStaticMarkup } from 'react-dom/server';
+import { Provider } from 'react-redux';
 /**
  * Internal Dependencies
  */
 import Gridicon from 'components/gridicon';
 import ContactFormDialog from './dialog';
+import { loadForm, addDefaultField, removeField, clearForm } from 'state/ui/editor/contact-form/actions';
+import { serialize, deserialize } from './shortcode-utils';
 
-const contactForm = editor => {
+const wpcomContactForm = editor => {
 	let node;
+	const store = editor.getParam( 'redux_store' );
```

Comment:
Ok, I tracked it down. 
Seems very different then what we do usually, but legit.